File: app/src/main/java/b2w/com/alodjinha/Model/Category.java
package b2w.com.alodjinha.Model;

public class Category {

    private String id;
    private String description;
    private String urlImage;


    public Category() {
    }

    public Category(String id, String urlImage, String description) {
        this.id = id;
        this.description = description;
        this.urlImage = urlImage;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getUrlImage() {
        return urlImage;
    }

    public void setUrlImage(String urlImage) {
        this.urlImage = urlImage;
    }
}

File: app/src/main/java/br/com/andrecouto/alodjinha/domain/model/lodjinha/Category.kt
package br.com.andrecouto.alodjinha.domain.model.lodjinha

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.SerializedName
import java.io.Serializable

@Entity(tableName = "categories")
data class Category(@SerializedName("id") @PrimaryKey var id : Int,
                    @SerializedName("descricao") var description : String?,
                    @SerializedName("urlImagem") var urlImage : String?) : Serializable